src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -214,6 +237,32 @@
       <Link>Common\Interop\Windows\SspiCli\SecPkgContext_StreamSizes.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="System\Net\Security\TlsCipherSuiteNameParser.ttinclude" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' and '$(AllowTlsCipherSuiteGeneration)' == 'true' ">
+    <Compile Include="System\Net\Security\TlsCipherSuiteData.Lookup.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>System\Net\Security\TlsCipherSuiteData.Lookup.tt</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' != 'true' and '$(AllowTlsCipherSuiteGeneration)' == 'true'">
+    <!-- Needed so that file can be generated on Windows from VS -->
+    <None Include="System\Net\Security\TlsCipherSuiteData.Lookup.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>TlsCipherSuiteData.Lookup.tt</DependentUpon>
+    </None>
+    <None Include="System\Net\Security\TlsCipherSuiteData.Lookup.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>TlsCipherSuiteData.Lookup.cs</LastGenOutput>
```

Comment:
A) Can this and the Unix one be unified?

B) Does it make sense to mark a DependentUpon on the `ttinclude`?

Reply:
A) Not quite (None vs Compile) but I was able to unify both template generators mostly so this logic should be simpler to follow
B) Unfortunatelly VS doesn't show it as expected so leaving as is